lib/dspace-item.ts

```diff
@@ -0,0 +1,60 @@
+import { Bitstream, Metadata } from './dspace-schemas';
+
+/**
+ * Returns an array of metadata items that match the given key.
+ * @param metadataItems - List of metadata to search.
+ * @param key - The key for the metadata items to find.
+ *
+ * @returns An array with metadata items for the given key.
+ * If no metadata items match the key an empty array is returned.
+ */
+export const findMetadataItems = (
```

Comment:
Where is this used?

Reply:
It's used in the indexer lambda that will have it's on PR as part of all of this refactoring.